src/main/java/edu/kis/vh/nursery/DefaultCountingOutRyhmer.java

```diff
@@ -44,4 +44,6 @@ public int getTotal() {
 //F3 (Or CTRL + B in IntelliJ) shows declaration of used method
 
 //Using Stack interface in constructor, instead of concrete classes lets utilize them interchangeably
-//without changing content of this class every time we want to use another implementation
\ No newline at end of file
+//without changing content of this class every time we want to use another implementation
+
+//All classes from this hierarchy break the isolation rule. Solution is to create common interface of these.
```

Comment:
Only one rhymer breaks isolation between abstraction and implementation hierarchies.